src/api/layers/GeoAdminAggregateLayer.class.js

```diff
@@ -109,7 +125,6 @@ export default class GeoAdminAggregateLayer extends GeoAdminLayer {
 
     clone() {
         let clone = super.clone()
-        clone.timeConfig = this.timeConfig.clone()
```

Comment:
This class inherit from `GeoAdminLayer` which potentialy has a `timeConfig`, removing this line could lead to subtle bugs in future ! For class completeness I would leave it but using `this.timeConfig?.clone() ?? null` instead. I know that for now having an aggregate with timeConfig doesn't make sense, but from a code perspective only it does.

Reply:
The cloning is managed by the parent class (inside parent's `.clone()` function, being called here with `super.clone()`), so we should be all good, no?

Reply:
Correct I missed that